### PR TITLE
Responsive Mobile Navbar with Mobile Menu for Personal Portfolio

### PR DESCRIPTION
**Pull request by Jhaman1895**
##In this milestone,  the following tasks were completed along with my partner Emmanuel[@Balikuddembe]:
  - When the user clicks (or taps) the hamburger button, the mobile menu appears.
  - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
  - When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
  - When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
  - Javascript linters were implemented for the first time.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,9 +11,20 @@
...
+    <div class="mobile-menu">
+      <div class="mobile-menu-close-toggle-container">
+        <a id="mobile-menu-close-toggle" onclick="toggleClasses()">
+          <img src="./images/icon_cancel.svg" alt="" class="mobile-menu-close-toggle-image">
+        </a>
+      </div>
+      <div class="mobile-menu-links">
+        <a href="#works" class="mobile-menu-links-items" onclick="toggleClasses()">Portfolio</a>
+        <a href="#aboutMe" class="mobile-menu-links-items" onclick="toggleClasses()">About</a><a href="#contact" class="mobile-menu-links-items" onclick="toggleClasses()">Contact</a>
+      </div>
```
Comment (zhadier) [inline index.html:23]: - [ ] Please ensure that the lines under the links span almost 80% of the mobile menu so it conforms to the Figma design

> Yours
image

> Figma
image

- [ ] Kindly also ensure that the gap between the links conform to the Figma design

- [ ]  Finally, please make sure that you add the white section at the end of your menu for stylistic conformation to the Figma design.
> Yours
image

> Figma
image

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @Jhaman1895  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Jhaman1895) [reply index.html]: Thank you for the suggestions. I have made changes in my code that I hope more closely match your suggestions.

**Review by rotimiazeez — APPROVED** [review]
# Hi TEAM 👏🏽 , 
## Good job making the suggested changes 👍🏽 
### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — COMMENTED** [review]
## Project Approved 👍 

Hi @Jhaman1895, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**